api.md

```diff
@@ -226,6 +229,14 @@ and return a positive `integer` hash value. Note that this hash value
 is _not_ a partition ID, but rather the inserted value's position in
 the dimension's key space, which is then divided across the partitions.
 
+[Adaptive chunking][adaptive-chunking] can be enabled by setting the
+`chunk_target_size` to a human-readable size, such as `'2GB'` or
```

Comment:
Yes, but you will get the following error if not:

```
select create_hypertable('cond', 'time', chunk_target_size => '2gb');
ERROR:  invalid data amount
HINT:  Valid units for this parameter are "kB", "MB", "GB", and "TB".
```

But maybe still worth mentioning upfront?

Reply:
Yah, it should be explicit in the docs I think.

Reply:
You could make a separate section for "Units" similar to the API entry for `create_hypertable` http://docs.timescale.com/v0.10/api#create_hypertable

Reply:
Thought I'd expand on the unit thing here.

As indicated in the PostgreSQL docs pointed to above, the "Memory With Unit" parameters (which is used by `chunk_target_size`) has a base unit, e.g., Kb (1024) or a block (8k). Currently, our base unit is one block (8k), which is the same as for the `shared_buffers` setting in PostgreSQL. 

This has the implication that if you'd set `chunk_target_size` to a "machine"-numeric value, such as 1, then the size in bytes would be 8k bytes (8 * 1024).

The reason for 8k block size is that the internal parsing function only accepts an `int` so in order to not overflow for large sizes, the base unit is 8k.

Maybe we should mention the base unit in case people input numerical values (probably not common). They might expect such a value to be in bytes and not blocks?

I chose to avoid talking about this since it is not the preferred way to input the `chunk_target_size` and the configuration page for `shared_buffers` isn't explicit about this either.

Reply:
Ah.  So it sounds like this also accepts plain numerics "8388608"? (1024*1024*8)  But in this case, rather than 8MB, this could actually be 64GB?   I think that mistake will be made by somebody, so we should definitely document.